Test/dafny0/ForallCompilationNewSyntax.dfy.expect

```diff
@@ -1,2 +1,6 @@
+ForallCompilationNewSyntax.dfy(26,4): Warning: /!\ No terms found to trigger on.
+ForallCompilationNewSyntax.dfy(35,4): Warning: /!\ No terms found to trigger on.
+ForallCompilationNewSyntax.dfy(44,4): Warning: /!\ No terms found to trigger on.
+ForallCompilationNewSyntax.dfy(64,4): Warning: /!\ No trigger covering all quantified variables found.
```

Comment:
I'd prefer to add triggers by hand instead of this, to avoid unrelated messages.  But this is fine.

Reply:
How do I do that? Or find documentation for how to do that?

Reply:
I think this:

```
  forall x: int <- {1, 2} {:trigger false} {    
  }
```

should work

Reply:
That gives a Boogie error
`ForallCompilationNewSyntax.dfy(26,4): Error: trigger must mention all quantified variables, but does not mention: k#0_2`

Reply:
Hmm, that's a regression.  Let's stick with what you have for now then.